### PR TITLE
Fix for WFMP-257 and WFMP-258

### PR DESCRIPTION
**Pull request by jfdenise**
Issues:
* https://issues.redhat.com/browse/WFMP-257
* https://issues.redhat.com/browse/WFMP-258

### CODE REVIEW

File: plugin/src/main/java/org/wildfly/plugin/provision/AbstractProvisionServerMojo.java
Comment (jamezp) [inline plugin/src/main/java/org/wildfly/plugin/provision/AbstractProvisionServerMojo.java:1]: None of the changes in this file should be needed.

File: plugin/src/main/java/org/wildfly/plugin/common/GlowMavenMessageWriter.java [hunk 1]
```diff
@@ -15,7 +15,7 @@ public class GlowMavenMessageWriter implements GlowMessageWriter {
 
     private final Log log;
 
-    GlowMavenMessageWriter(Log log) {
+    public GlowMavenMessageWriter(Log log) {
```
Comment (jamezp) [inline plugin/src/main/java/org/wildfly/plugin/common/GlowMavenMessageWriter.java:18]: This change does not appear to be needed.

File: plugin/src/main/java/org/wildfly/plugin/provision/AbstractPackageServerMojo.java [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+    protected boolean skip;
+
+    /**
+     * Skip deploying the deployment after the server is provisioned ({@code false} by default).
+     */
+    @Parameter(defaultValue = "false", property = PropertyNames.SKIP_PACKAGE_DEPLOYMENT)
+    protected boolean skipDeployment;
+
+    @Inject
+    protected OfflineCommandExecutor commandExecutor;
```
Comment (jamezp) [inline plugin/src/main/java/org/wildfly/plugin/provision/AbstractPackageServerMojo.java:167]: This can be private.

**Review by jamezp — CHANGES_REQUESTED** [review]
Could I get a little more information on why this change is needed? The JIRA's don't really say much :)

**Comment by jfdenise** [comment]
Closing this PR, will open a PR for WFMP-257 only, WFMP-258 will be not done.

**Pull request closed without merging**